git_machete/git_operations.py

```diff
@@ -918,7 +918,10 @@ def do_rebase() -> None:
             try:
                 if not opt_no_interactive_rebase:
                     rebase_opts.append("--interactive")
-                self._run_git("rebase", *rebase_opts, "--empty=drop", "--onto", onto, from_exclusive, branch)
+                if self.get_git_version() < (2, 26, 0):
```

Comment:
Nit: commit message is incorrect:

<img width="298" alt="image" src="https://user-images.githubusercontent.com/3383210/229093727-dc1e2f48-4a2e-4127-82da-9bfce9dacfa3.png">

Should be `>=`, not `>`